cloud_chatops/compose.yaml

```diff
@@ -0,0 +1,11 @@
+services:
+  cloud_chatops:
+    image: harbor.stfc.ac.uk/stfc-cloud/cloud-chatops
+    volumes:
+      - ./cloud_chatops_secrets/:/opt/cloud_chatops_secrets/
```

Comment:
This should either use an absolute path or a dedicated Docker Compose volume

Because it's relative the effects can change (e.g. if you run `docker-compose up` from another folder)